Tropos/Sources/Controllers/AppDelegate.swift

```diff
@@ -0,0 +1,70 @@
+import HockeySDK
+import os.log
+import TroposCore
+import UIKit
+
+@UIApplicationMain
+final class AppDelegate: UIResponder, UIApplicationDelegate {
+    let applicationController = ApplicationController()
+    var window: UIWindow?
+
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions _: [UIApplicationLaunchOptionsKey: Any]? = nil
+    ) -> Bool {
```

Comment:
This function is _giant_. Could it be broken down?

Reply:
Sure, we could extract some setup functions.

Reply:
Extracted `setupHockey()` in 489f7f1